src/v3/test/vrt/start-app-vrt.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export NODE_OPTIONS="--max-old-space-size=5120"
+yarn dev
```

Comment:
We should be able to use the other script: `test/e2e/start-app-e2e.sh` and remove this one since they are identical